docs/source/contributing/tests.rst

```diff
@@ -4,19 +4,16 @@
 Testing JupyterHub and linting code
 ===================================
 
-Unit test help validate that JupyterHub works the way we think it does,
-and continues to do so when changes occur. They also help communicate
-precisely what we expect our code to do. 
+Unit tests help confirming that JupyterHub works as intended and does so even when modifications are made. Additionally, they help in clarifying our expectations for our code.
 
-JupyterHub uses `pytest <https://pytest.org>`_ for all our tests. You
-can find them under ``jupyterhub/tests`` directory in the git repository.
+JupyterHub uses `pytest <https://pytest.org>`_ for all the tests. You
+can find them under the `jupyterhub/tests <https://github.com/jupyterhub/jupyterhub/tree/main/jupyterhub/tests>`_ directory in the git repository.
 
 Running the tests
 ==================
 
-#. Make sure you have completed :ref:`contributing/setup`. You should be able
-   to start ``jupyterhub`` from the commandline & access it from your
-   web browser. This ensures that the dev environment is properly set
+#. Make sure you have completed :ref:`contributing/setup`. Once completed, you should be able
+   to start ``jupyterhub`` through a web browser as well as the command line. By doing this, it is ensured that the dev environment is properly set
```

Comment:
Do you mean you can open JupyterHub in a web browser after starting it on the command line?

Reply:
@manics I guess that is the case. Since step 7 and 8 in the set up docs state

"
 7. You are now ready to start JupyterHub!
``jupyterhub``
8. You can access JupyterHub from your browser at http://localhost:8000 now.
"